XCTWebDriverAgentLib/Commands/FBFindElementCommands.m

```diff
@@ -206,6 +208,13 @@ + (void)descendantsOfElement:(XCUIElement *)element withProperty:(NSString *)pro
   [results addObjectsFromArray:childElements];
 }
 
+#pragma mark - Search by Predicate String
++ (NSArray *)descendantsOfElement:(XCUIElement *)element withPredicateString:(NSString *)predicateString {
+    XCUIElementQuery *query = [[element descendantsMatchingType:XCUIElementTypeAny] matchingPredicate:[NSPredicate predicateWithFormat:predicateString]];
```

Comment:
I guess it will not work with UIAutomation class names? Which we should remove anyway ;).